src/applications/vaos/components/FacilityAddress.jsx

```diff
@@ -7,17 +8,24 @@ export default function FacilityAddress({
   name,
   facility,
   showDirectionsLink,
-  isHomepageRefresh,
-  clinicFriendlyName,
+  clinicName,
```

Comment:
The switch in here was really just to show or not show the clinic name, so rather than use a homepage refresh prop, I made that part conditional on the clinic name prop. Both use cases should be marked up with real heading tags, instead of one using strong tags and the other using a definition list